docs/next-steps/reset.md

```diff
@@ -0,0 +1,17 @@
+# Resetting Your Data
+
+The reset button gives you a blank slate, of sorts, to perform a fresh new sync. This can be useful if you are just testing Airbyte or don't necessarily require the data replicated to your destination to be saved permanently.
+
+![](../.gitbook/assets/reset_your_data_1.png)
+
+As outlined above, you can click on the `Reset your data` button to give you that clean slate. Just as a heads up, here is what it does and doesn't do:
+
+The reset button **DOES**:
+- Delete all records in your destination tables
+- Delete all records in your destination file
```

Comment:
@tuliren is this true for S3?

Reply:
Sorry that I missed this question. (Just created dedicated email filters so that direct message won't get buried in the inbox.)

Yes, this is true. Reset trigger a full refresh - overwrite, which will purge the configured bucket path. I have verified this behavior.